cypress/integration/pages/testsForAllPages.js

```diff
@@ -230,15 +230,34 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
     });
 
     describe('Header Tests', () => {
+      const hasLocalisedName =
+        appConfig[service][variant].serviceLocalizedName !== undefined;
+
       it('should render the BBC News branding', () => {
         cy.get('header a').should(
           'contain',
-          appConfig[service][variant].serviceLocalizedName !== undefined
+          hasLocalisedName
             ? `${appConfig[service][variant].product}, ${appConfig[service][variant].serviceLocalizedName}`
             : appConfig[service][variant].product,
         );
       });
 
+      if (hasLocalisedName) {
+        it("should have offscreen text with product's language code set to English", () => {
+          cy.get('header a span span').should('have.attr', 'lang', 'en-GB');
```

Comment:
It'd be good to be more specific, stating that we're looking at the visually hidden text within the Banner.

```suggestion
          cy.get('header div[class^="Banner"] span[class^="VisuallyHiddenText"] span').should('have.attr', 'lang', 'en-GB');
```